packages/@webex/plugin-cc/src/services/core/HttpRequest.ts

```diff
@@ -31,58 +15,6 @@ class HttpRequest {
   private constructor(options: {webex: WebexSDK}) {
     const {webex} = options;
     this.webex = webex;
-    this.webSocket = new WebSocket({
-      parent: this.webex,
-    });
-    this.eventHandlers = new Map();
-
-    // Centralized WebSocket event listener
-    this.webSocket.on(EVENT, (eventData) => {
-      this.webex.logger.log(`Received event: ${eventData.type}`);
-      const handler = this.eventHandlers.get(eventData.type);
-      if (handler) {
-        handler(eventData.data);
-      }
-    });
-  }
-
-  public getWebSocket(): IWebSocket {
-    return this.webSocket;
-  }
-
-  /* This calls subscribeNotifications and establishes a websocket connection
-   * It sends the request and then listens for the Welcome event
-   * If the Welcome event is received, it resolves the promise
-   * If the Welcome event is not received, it rejects the promise
-   */
-  public async subscribeNotifications(options: {body: SubscribeRequest}): Promise<WelcomeResponse> {
-    const {body} = options;
-    const eventType = CC_EVENTS.WELCOME;
-    const subscribeResponse: SubscribeResponse = await this.webex.request({
-      service: WCC_API_GATEWAY,
-      resource: SUBSCRIBE_API,
-      method: HTTP_METHODS.POST,
-      body,
-    });
-
-    return new Promise((resolve, reject) => {
-      const timeoutId = setTimeout(() => {
-        this.webex.logger.error('Timeout waiting for event');
-        this.eventHandlers.delete(eventType);
-        reject(new Error('Timeout waiting for event'));
-      }, WEBSOCKET_EVENT_TIMEOUT);
-
-      // Store the event handler
-      this.eventHandlers.set(eventType, (data: WelcomeEvent) => {
-        clearTimeout(timeoutId);
-        this.eventHandlers.delete(eventType);
-        resolve(data);
-      });
-
-      this.webSocket.connectWebSocket({
-        webSocketUrl: subscribeResponse.body.webSocketUrl,
-      });
-    });
   }
 
   public async request(options: {
```

Comment:
We only need to retain this method, the rest of them are now supported via the `WebSocketManager`.
Discussed with Ravi and we felt this was the best course of action.